scholia/query.py

```diff
@@ -809,6 +809,107 @@ def q_to_label(q, language='en'):
         return None
 
 
+def q_to_label_and_description(q, language='en'):
+    """Get label and description for Q item.
+
+    Parameters
+    ----------
+    q : str
+        String with Wikidata Q item.
+    language : str
+        String with language identifier
+
+    Returns
+    -------
+    label : str
```

Comment:
this documentation style is more appropriate for when you return a 2-tuple instead of a dictionary. Maybe using a `namedtuple` would even be best here!

Reply:
ah, good catch. copy/paste artifact.